gpflow/logdensities.py

```diff
@@ -85,13 +82,6 @@ def multivariate_normal(x, mu, L):
     p[n] = log pdf of:
     x[n] ~ N(mu, LL^T) or x ~ N(mu[n], LL^T) or x[n] ~ N(mu[n], LL^T)
     """
-    mu_ndims = mu.shape.ndims
-    x_ndims = x.shape.ndims
-    if x_ndims is not None and x_ndims != 2:
```

Comment:
Please, leave this file without change. Warnings removed in develop. Exceptions should stay.

Reply:
If you want an exception it should be a tf.Assert, no?

Reply:
No, tf.Assert is a run-time, I want it on graph building stage.

Reply:
why do you even care? Alessandro added the check, and it only gets used in GPR anyways ...